commands/add.go

```diff
@@ -15,7 +15,7 @@ const (
 )
 
 func init() {
-	RootCmd.AddCommand(&cobra.Command{
+	rootCmd.AddCommand(&cobra.Command{
```

Comment:
Yeah... it got lumped into the cleanup of _things being exported that shouldn't be exported_.